### DISCUSSION TITLE
Markdown table containing implicitly updated dependency information is not rendering properly on bitbucket

### DISCUSSION THREAD

**Discussion by hrivera-ntap** [renovatebot/renovate#29612] [Suggest an Idea]
### Tell us more.

The change (#28938) to add a comment when golang updates implicitly update indirect dependencies does not properly render the markdown table on bitbucket platforms, see the below image:

<img width="1005" alt="image" src="https://github.com/renovatebot/renovate/assets/70445595/6374b998-cc21-4c75-aff2-9c998d0d8f54">

I modified the comment to include an extra newline break after the word `Details:` and it appeared to fix it.

**Comment by rarkins** [comment]
https://github.com/renovatebot/renovate/pull/29615

**Reply by hrivera-ntap** [reply]
Thanks for the quick resolution @rarkins!
